docs/source/narrative/contributing/develop.rst

```diff
@@ -8,40 +8,61 @@ Developing Websauna
 Prerequisites
 -------------
 
-* GCC, make, and similar (``apt-get install build-essential``)
-* Python 3.5 with development headers (``apt-get install python-dev``)
-* virtualenv (``apt-get install python-virtualenv``)
-* pip (``apt-get install python-pip``)
-* git (``apt-get install git``)
-* PhantomJS headless browser (``apt-get install phantomjs``)
-* Redis (``apt-get install redis``)
+In order to develop Websauna, the following requirements need to be installed on your system:
+
+    * GCC, make, and similar
+    * Python 3.5 with development headers
```

Comment:
"or above" ?

Reply:
Not yet. We are still supporting only 3.4 and 3.5 (even though in the Pyramid 1.9 branch I added 3.6 support as well).